roles/branch-validation/tasks/main.yml

```diff
@@ -100,71 +101,34 @@
     ansible_ssh_host: "{{ fip.floating_ip.floating_ip_address }}"
 
 ### Node connectivity tests
-- name: cross-node inter-instance connectivity test
-  local_action:
-    module: "command ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
-            -o IdentityFile={{ temp_key_dest }}
-            cirros@{{ fip.floating_ip.floating_ip_address }}
-            ping -c 5 {{ item.openstack.private_v4 }}"
-  sudo: no
-  changed_when: false
-  register: pings
-  until: pings|success
-  delay: 1
-  retries: 5
-  with_items: instances.results
 
-- name: instance internet connectivity test w/ floating-ip
-  local_action:
-    module: "command ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
-            -o IdentityFile={{ temp_key_dest }}
-            cirros@{{ fip.floating_ip.floating_ip_address }}
-            ping -c 5 google.com"
-  sudo: no
-  changed_when: false
-  register: pings
-  until: pings|success
-  delay: 1
-  retries: 5
-
-- name: wait for user to login
-  pause: prompt="From {{ groups['controller'][0] }} ssh -o
-                 UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
-                 -o IdentityFile={{ temp_key_dest }}
-                 cirros@{{ fip.floating_ip.floating_ip_address }}
-                 (press enter when done)"
-
-- name: disassociate the floating ip
-  os_floating_ip:
-    auth:
-      username: admin
-      project_name: admin
-      password: "{{ secrets.admin_password }}"
-      auth_url: "http://127.0.0.1:{{ endpoints.keystone_admin.port.backend_api }}/{{ endpoints.keystone.version }}"
-    network: external
-    server: "validate-{{ groups[instance_group][0] }}"
-    floating_ip_address: "{{ fip.floating_ip.floating_ip_address }}"
-    state: absent
```

Comment:
Even though there's no "commented on outdated diff" message, rest assured this is addressed here: https://github.com/angstwad/ursula/blob/152e896be75284075b9d0a79dbe10f83f0edfdbc/roles/branch-validation/tasks/main.yml#L144-L165